spec/doorkeeper-jwt/doorkeeper-jwt_spec.rb

```diff
@@ -55,7 +55,7 @@
       end
 
       token = Doorkeeper::JWT.generate({})
-      decoded_token = ::JWT.decode(token, "super secret", "HS256")
+      decoded_token = ::JWT.decode(token, "super secret", true, {algorithm: "HS256"})
```

Comment:
Line is too long. [85/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.